File: app/src/main/java/closer/vlllage/com/closer/handler/helpers/DateFormatter.java
package closer.vlllage.com.closer.handler.helpers;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;
import java.util.TimeZone;

import closer.vlllage.com.closer.pool.PoolMember;

public class DateFormatter extends PoolMember {

    private final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);

    public String format(Date date) {
        dateFormat.setTimeZone(TimeZone.getTimeZone("UTC"));
        return dateFormat.format(date);
    }
}

File: app/src/main/java/closer/vlllage/com/closer/handler/helpers/DateFormatter.kt
package closer.vlllage.com.closer.handler.helpers

import com.queatz.on.On
import java.text.SimpleDateFormat
import java.util.*

class DateFormatter constructor(private val on: On) {

    private val dateFormat = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US)

    fun format(date: Date): String {
        dateFormat.timeZone = TimeZone.getTimeZone("UTC")
        return dateFormat.format(date)
    }
}
